tests/util/util_bucket.sh

```diff
@@ -247,12 +247,12 @@ bucket_cleanup_if_bucket_exists() {
 # params:  client, bucket name(s)
 # return 0 for success, 1 for failure
 setup_buckets() {
-  if [ $# -lt 2 ]; then
-    log 2 "'setup_buckets' command requires client, bucket names"
+  if [ $# -lt 1 ]; then
```

Comment:
Could use check_param_count here?

Reply:
check_param_count was intended by me for "run/assert" combinations on common test procedures but with different clients (e.g. in test_common.sh), so that I can return a proper error messages for param count errors there.  I added more of these checks there though.

Reply:
Yeah, the checks look good.  Just figured that it could also be used within the test functions, since it doesn't have run/assert in it it wouldn't disrupt error returns but it would provide better error messaging.

Reply:
Looked at it again, yeah, it could eliminate a line